weasyl/controllers/api.py

```diff
@@ -152,6 +152,23 @@ def tidy_submission(submission):
     return submission
 
 
+def get_tidy_journal(journal):
+    new_journal = {x: journal[x] for x in journal if x in ["title", "content", "journalid", "tags"]}
+    new_journal['welcomeid'] = journal['id']
+    new_journal['posted_at'] = d.iso8601(journal['unixtime'])
+    contype = journal['contype'] if 'contype' in journal else None
```

Comment:
Don’t journal dicts coming into this function always have a `'contype'` (that’s always 30)? Might be best not to include it at all.